LayoutTests/resources/testharnessreport.js

```diff
@@ -61,6 +61,7 @@ if (self.testRunner) {
 
     // window.opener is a configurable property, so store it before we run anything.
     const orig_opener = window.opener;
+    const isInCrossOriginFrame = window.location.href.indexOf("inCrossOriginFrame=true") >= 0;
```

Comment:
Could we check href.searchParams.has("inCrossOriginFrame", "true")?
Also, could we make this even more explicit "inCrossOriginFrameForSiteIsolationTesting" or something?